src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const listElement = document.querySelector('ul');
+const employeesArr = [...listElement.children];
+
+listElement.innerHTML = '';
+
+sortWorkers(employeesArr);
+getEmployees(employeesArr);
+
+function sortWorkers(arr) {
+  const sortedWorkers = arr.sort((employee1, employee2) => {
+    const salary1 = +employee1.dataset.salary.replace(/[$,]/g, '');
+    const salary2 = +employee2.dataset.salary.replace(/[$,]/g, '');
+
+    return salary2 - salary1;
+  });
+
+  listElement.append(...sortedWorkers);
+}
```

Comment:
The task description specifies a function named `sortList` that should take a `list` property. Consider renaming `sortWorkers` to `sortList` and ensuring it takes a list as an argument, as per the task requirements.